crates/ruff_linter/src/rules/isort/snapshots/ruff_linter__rules__isort__tests__skip.py.snap

```diff
@@ -52,9 +51,10 @@ skip.py:27:1: I001 [*] Import block is un-sorted or un-formatted
 
 skip.py:34:1: I001 [*] Import block is un-sorted or un-formatted
    |
-32 |       import sys; import os  # isort:skip
-33 |       import sys; import os  # isort:skip  # isort:skip
-34 | /     import sys; import os
+32 |     import sys; import os  # isort:skip
+33 |     import sys; import os  # isort:skip  # isort:skip
+34 |     import sys; import os
```

Comment:
Agree, this seems correct and is more a problem with `I001` then the rendering (unless `I001` generates the correct range